REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 0.7
 Distributions
 StatsBase
-Compat 0.27
+Compat 0.50
```

Comment:
You can actually drop the Compat dependency now that 0.7 is required.

Reply:
Thanks, good call. Fixed